src/riak_kv_entropy_manager.erl

```diff
@@ -219,6 +221,20 @@ clear_trees() ->
 get_version() ->
     gen_server:call(?MODULE, get_version, infinity).
 
+-spec get_partition_version(index()) -> version().
+get_partition_version(Index) ->
+    try
+        case ets:lookup(?ETS, Index) of
+            [] ->
+                legacy;
+            [{Index, Version}] ->
+                Version
+        end
+    catch
```

Comment:
In looking around at how this was implemented in other places, I came across the riak_core_capability:get function which uses this try catch. I also looked at the docs and didn't understand why it was there and also looked in the commit history for capabilties and didn't find anything. Unssure of if I was missing something, I just added the try catch because there was no downside from a logic perspective. With that said, I agree and will remove the try/catch as I havent found a single reason why it's there.